components/common/SearchBox/SearchInput.tsx

```diff
@@ -18,24 +22,67 @@ function SearchInput() {
     }
   };
 
+  const handlePaste = async (event: MouseEvent<HTMLDivElement>) => {
+    setIsFocused(true);
+    event.preventDefault();
+    const pastedText = await navigator.clipboard.readText();
+    setQuery(pastedText);
+  };
+
+  const handleInputFocus = () => {
+    setIsFocused(true);
+  };
+
+  const handleInputBlur = () => {
+    setTimeout(() => {
+      setIsFocused(false);
+    }, BLUR_DELAY);
+  };
   return (
     <form
       onSubmit={onSubmit}
-      className="bg-none md:bg-neutral-500 w-full md:w-[66%] md:py-8 flex justify-center md:justify-start items-center rounded-soft flex-col md:flex-row ">
-      <div className="w-full bg-neutral-500 md:bg-none p-12 rounded-soft md:rounded-none md:py-0 md:px-25 flex justify-center items-center">
-        <SearchIcon className="mr-5 text-neutral-400" />
+      className={`bg-none md:bg-neutral-500 flex justify-center md:justify-start items-center ${
+        roundedFull
+          ? 'rounded-full py-2.5 w-full'
+          : 'rounded-soft md:py-8 w-full md:w-[66%]'
+      } flex-col md:flex-row`}>
+      <div
+        className={`w-full bg-neutral-500 md:bg-none  
+      ${
+        roundedFull
+          ? 'rounded-full px-2.5'
+          : 'rounded-soft md:rounded-none p-12 md:py-0 md:px-25'
+      }
+       flex justify-center items-center`}>
+        <SearchIcon size="1.25rem" className="mr-5 text-neutral-400" />
         <input
+          onBlur={handleInputBlur}
+          onFocus={handleInputFocus}
           className="w-full text-neutral-200 bg-neutral-500 focus:outline-0 overflow-hidden text-14 md:text-16"
           placeholder="Search by Request ID, Wallet or Tx Hash"
```

Comment:
![image](https://github.com/rango-exchange/explorer/assets/141495369/a4fa8a3b-9811-4df9-a6a6-55812b0fb6ec)

remove place holder, when is focused.

Reply:
fixed